radicle/src/node.rs

```diff
@@ -180,6 +181,63 @@ impl Command {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, serde::Serialize, serde::Deserialize)]
+#[serde(tag = "type", content = "id")]
```

Comment:
```suggestion
#[serde(tag = "state", content = "id")]
```
Would be my suggestion, since it's the state of the node session, not the type of node. I'd also make sure the tags are kebab-case, which we do everywhere else for the control socket json.